public/index.html

```diff
@@ -280,6 +280,7 @@ <h3>Your conversion data</h3>
                         <div class="help" id="help-daily"></div>
                         <label for="zero-pct"> Percentage of buckets with 0 conversions: </label>
                         <input name="zero-pct" id="zero-pct" type="number" value="0" min="0" max="10"></input>
+                        <div class="help" id="help-zero-pct"></div>
```

Comment:
Looks good!
Suggestion: also add a similar hint as for epsilon 
`<div class="input-hint below-input">Max: 10%</div>`